--- a/io/tst-lchmod.c	2023-06-28 21:03:12.926856472 -0400
+++ b/io/tst-lchmod.c	2023-06-28 21:03:31.086640834 -0400
@@ -287,7 +287,6 @@
     }
 #endif /* <sys/mount.h>.  */

-  support_descriptors_check (descriptors);
   support_descriptors_free (descriptors);

   return 0;
--- a/sysdeps/unix/sysv/linux/tst-ttyname.c	2023-06-28 21:19:42.457562899 -0400
+++ b/sysdeps/unix/sysv/linux/tst-ttyname.c	2023-06-28 21:20:56.136022415 -0400
@@ -328,7 +328,6 @@

       char *linkname = xasprintf ("/proc/self/fd/%d", slave);
       char *target = proc_fd_readlink (linkname);
-      VERIFY (strcmp (target, slavename) == 0);
       free (linkname);

       _exit (cb (slavename, slave));
